doc/examples.xml

```diff
@@ -529,6 +574,51 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="RooksGraph">
+<ManSection>
+  <Oper Name="RooksGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers, then this
+    operation returns the <E>rook's graph</E> for an <A>m</A> 
+    by <A>n</A> board.<P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/RookGraph.html</URL>:
+    <P/>
+
+    <Q>The <C><A>m</A> x <A>m</A></C> rook's graph (confusingly called the 
+    <C><A>m</A> x <A>m</A></C> grid by Brouwer et al.1989, p. 440) and also
+    sometimes known as a lattice graph (e.g., Brouwer) is the graph Cartesian 
+    product <C>K_m square K_n</C> of complete graphs, which is
```

Comment:
I guess it would probably make more sense to prioritize the aesthetics of the PDF over the aesthetics of the plain text manual, so that sounds good to me!